src/days-2-4-adv/adv.py

```diff
@@ -49,3 +50,28 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+while True:
+    print (p.__str__())
```

Comment:
If you give your player object a room object, you could then call the name and description `print (p.room.name, p.room.description)`